src/appleseed.python/bindimage.cpp

```diff
@@ -61,61 +61,73 @@ namespace
         return new Tile(*source);
     }
 
-    void copy_tile_data_to_py_buffer(const Tile& tile, bpy::object& buffer)
+    Image* copy_image(const Image* source)
```

Comment:
(I can't comment on the proper line.) `<algorithm>` appears to no longer be needed? Check the other header files too.